xml/pmo/resumes/-duplicates.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2016-2018 Zerocracy
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to read
+the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+merge, publish, distribute, sublicense, and/or sell copies of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<resumes version="1" updated="2016-12-29T09:03:21.684Z">
+  <resume login="yegor256">
+    <submitted>2018-01-01T00:00:00</submitted>
+    <examiner>g4s8</examiner>
+    <text>I'm great</text>
+    <personality>INTJ-A</personality>
+    <stackoverflow>187141</stackoverflow>
+    <telegram>yegor256</telegram>
+  </resume>
+  <resume login="yegor256">
+    <submitted>2018-01-01T00:00:00</submitted>
+    <examiner>g4s8</examiner>
```

Comment:
@g4s8 since we're enforcing uniqueness of the login attribute, let's make the rest of data different.